src/pypgstac/pyproject.toml

```diff
@@ -46,7 +46,7 @@ dev = [
     "black>=21.7b0",
     "mypy>=0.910",
     "types-orjson==0.1.1",
-    "types-pkg-resources",
+    "types-setuptools",
```

Comment:
I got an error when I built the docker containers saying `types-pkg-resources` had been "pulled" and to replace it with `types-setuptools`